app/model/Travel/Command/TransportTravel.php

```diff
@@ -17,9 +17,9 @@
 class TransportTravel extends Travel
 {
     /**
-     * @ORM\Column(type="money", name="distance")
+     * @ORM\Column(type="float", name="distance")
```

Comment:
proč float kdyz to ma byt int?

Reply:
To přemapování sloupců je v další migraci.